DEVDOC.md

```diff
@@ -396,3 +396,265 @@ bazel-bin/gapii/cc/vulkan_spy_1.cpp
 To get a unified file tree view, interleaving source files with generated files,
 you can use the `cmd/gofuse` setup. `cmd/gofuse` links the generated source
 files in the same directories as their siblings in the same packages/namespaces.
+
+-------------------------------------------------------------------------------
+
+## Life of a gfxtrace
+
+This is an overview of how AGI captures graphics API calls, stores them into a
+gfxtrace file, and process that file to replay the calls.
+
+This is oriented to AGI developers who need to work with AGI internals. It is
+meant to help build a high-level mental model that facilitates navigating the
+actual source code. It references concepts and code modules that are unlikely to
+change anytime soon, and it tries not to reference details that are likely to
+change. You are encouraged to check the source code while reading this, to
+clarify how the various parts actually fits together. If you spot any
+discrepancy, please update this doc, but please keep it high-level.
+
+AGI is architectured to be API-agnostic in order to support multiple graphics
+API. GAPID, from which AGI was forked, supports GLES and Vulkan. AGI main focus
```

Comment:
APIs should be plural. And I think better use "AGI's main focus", but better double check with a native English speaker.

Reply:
done